instrumentation/othttp/common.go

```diff
@@ -28,6 +28,14 @@ const (
 	WriteErrorKey = kv.Key("http.write_error") // if an error occurred while writing a reply, the string of the error (io.EOF is not recorded)
 )
 
+// Server HTTP metrics
+const (
+	RequestCount  = "request_count"  // request count total
+	RequestBytes  = "request_bytes"  // request bytes total
+	ResponseBytes = "response_bytes" // response bytes
+	ServerLatency = "server_latency" // End to end latency
```

Comment:
Can we use `http.server.duration` here to try to get out in front of the [metric semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/pull/657/files#diff-1bebd1c3cbc1c765fff51ff821155a1fR21) that are in flight?

Reply:
done